device-types/HPE/DX380_Gen10.yaml

```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: HPE
+model: DX380 Gen10
+slug: hpe_dx380_g10

```

Comment:
Separate out into a different PR

Looks like you may be missing your module for PCI risers as well